dart/example/main.dart

```diff
@@ -5,7 +5,10 @@
 import 'dart:async';
 import 'dart:io';
 
-import 'package:sentry/sentry.dart';
+import 'package:sentry/src/sentry.dart';
+import 'package:sentry/src/sentry_options.dart';
```

Comment:
ideally `sentry_options.dart` is already imported by `sentry.dart`

Reply:
@rxlabz any reason to ignore this one?

Reply:
no it was updated

Reply:
ah ok, sorry, I used to let the issue opener close it, but I'll close it when I'm sure it was resolved
.

Reply:
no worries, I also close them when I see it, but if there are no comments or reactions, I think this one should have been closed by me btw haha